atest/testdata/standard_libraries/string/convert_to.robot

```diff
@@ -29,3 +52,11 @@ Do lowercase
     [Arguments]    ${arg1}    ${arg2}
     ${result} =    Convert To Lowercase    ${arg1}
     Should be Equal    ${result}    ${arg2}
+
+Do titlecases
+    [Arguments]    ${arg1}    ${arg2}
+    ${result} =    Convert To Titlecase    ${arg1}
+    Should be Equal    ${result}    ${arg2}
+
+
+	
```

Comment:
Please remove unnecessary empty lines. Yes, we are pretty picky about using whitespace.

Reply:
Kept only one empty line at the end of the file.